views/helpers/annotations.js

```diff
@@ -0,0 +1,157 @@
+const extent = require("../helpers/extent.js");
+
+/**
+ * Returns true, if there is at least one annotation on the left or on the right.
+ */
+function hasAnnotationOnLeftOrRight(annotations, cssModifier) {
+  if (cssModifier === "narrow") return false; // on mobile...
```

Comment:
Rather than adding the comment "on mobile..." you could add here what will happen if mobile view is shown. E.g. 

Returns true, if there is at least one annotation on the left or on the right. On mobile, annotations on the left or right won't be shown.

Or whatever is the case.